ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -1045,6 +1046,24 @@ private void registerCommandTopic() {
 
     final String commandTopic = commandStore.getCommandTopicName();
 
+    // migration code
```

Comment:
do we know that this will run for only one pod at a time?

Reply:
I think it's possible that this could run in multiple pods, I think with the addition of the transaction, we may end up in a situation where multiple pods are fighting for the migration potentially.